### DISCUSSION TITLE
3.0.6

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-core#5015] [Design & tech]
- [Prevent deletion of all records with empty conditions](https://github.com/elsa-workflows/elsa-core/commit/12de2902870a776121c480edd3b176169365ef23)
- [Fix Security Bug: No permissions applied to version endpoints (](https://github.com/elsa-workflows/elsa-core/commit/875afd4250b34b329772afce0cb33365f8180119)https://github.com/elsa-workflows/elsa-core/pull/5008[)](https://github.com/elsa-workflows/elsa-core/commit/875afd4250b34b329772afce0cb33365f8180119)

**Full Changelog**: https://github.com/elsa-workflows/elsa-core/compare/3.0.5...3.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-core/releases/tag/3.0.6'>3.0.6</a>.</em>

**Comment by glime-ai** [comment]
It seems like you are referencing two commits from the Elsa Workflows GitHub repository related to preventing the deletion of all records with empty conditions and fixing a security bug where no permissions were applied to version endpoints. These changes are part of the full changelog for the release tagged 3.0.6.

To view the details of these commits and the full changelog, you can visit the release page for 3.0.6 on the Elsa Workflows GitHub repository at the following URL: [Elsa 3.0.6 Release](https://github.com/elsa-workflows/elsa-core/releases/tag/3.0.6).

The specific commits you mentioned are:
- Prevent deletion of all records with empty conditions: [Commit 12de290](https://github.com/elsa-workflows/elsa-core/commit/12de2902870a776121c480edd3b176169365ef23)
- Fix Security Bug: No permissions applied to version endpoints: [Commit 875afd4](https://github.com/elsa-workflows/elsa-core/commit/875afd4250b34b329772afce0cb33365f8180119) and [Pull Request #5008](https://github.com/elsa-workflows/elsa-core/pull/5008)

Please let me know if you need further assistance or more detailed information about these changes.
